tests/e2e/graphql/workloads.yaml

```diff
@@ -0,0 +1,14 @@
+template_cluster: &template_cluster
+  cluster:
+    main:
+      args: ["--bolt-port", "7687", "--log-level", "TRACE", "--callable_mappings_path", "graphql/callable_alias_mapping.json"]
+      log_file: "graphql-e2e.log"
+      setup_queries: []
+      validation_queries: []
+
+workloads:
```

Comment:
Can you also add a workload for disk storage? You can check how is done [here](https://github.com/memgraph/memgraph/pull/1107). Just so we don't need to open later another PR